internal/app/tfsec/checks/aws019.go

```diff
@@ -41,13 +41,14 @@ func init() {
 		RequiredTypes:  []string{"resource"},
 		RequiredLabels: []string{"aws_kms_key"},
 		CheckFunc: func(check *scanner.Check, block *parser.Block, _ *scanner.Context) []scanner.Result {
-			keyRotationAttr := block.GetAttribute("enable_key_rotation")
 			keyUsageAttr := block.GetAttribute("key_usage")
 
-			if keyUsageAttr.Value().AsString() == "SIGN_VERIFY" {
+			if keyUsageAttr != nil && keyUsageAttr.Value().AsString() == "SIGN_VERIFY" {
```

Comment:
```suggestion
			if keyUsageAttr != nil && keyUsageAttr.Equals("SIGN_VERIFY") {
```

there are a number of helper functions that aim to keep these checks concise